shell/mixins/brand.js

```diff
@@ -159,47 +166,26 @@ export default {
       for (const prop in vars) {
         document.body.style.removeProperty(prop);
       }
-    }
-  },
-  head() {
-    let cssClass = `overflow-hidden dashboard-body`;
-    const out = {
-      bodyAttrs: { class: `theme-${ this.theme } ${ cssClass }` },
-      title:     getVendor(),
-    };
-
-    if (getVendor() === 'Harvester') {
-      const ico = require(`~shell/assets/images/pl/harvester.png`);
-
-      out.title = 'Harvester';
-      out.link = [{
-        hid:  'icon',
-        rel:  'icon',
-        type: 'image/x-icon',
-        href: ico
-      }];
-    }
-
-    let brandMeta;
-
-    if ( this.brand ) {
-      try {
-        brandMeta = require(`~shell/assets/brand/${ this.brand }/metadata.json`);
-      } catch {
-        return out;
+    },
+    setBodyClass() {
+      const body = document.getElementsByTagName('body')[0];
+      const cssClass = `overflow-hidden dashboard-body`;
+      let bodyClass = `theme-${ this.theme } ${ cssClass }`;
```

Comment:
I think you still need to set `document.title` to `getVendor()` as part of this process: it ensures that the title is "Rancher Prime" when the ui-brand global setting indicates that this is a paid version of the dash and a custom private label hasn't been set. Set ui-brand to 'suse' to check this behaviour.